docs/whatsnew/0.9.0.rst

```diff
@@ -0,0 +1,5 @@
+Version v0.9.0 (TBD)
+====================
+
```

Comment:
```suggestion
```

`towncrier` should auto-generated the title based on the toml configuration....At least, that's how we had it setup in the past.